pacta.executive.summary.Rproj

```diff
@@ -20,6 +20,7 @@ BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
 PackageRoxygenize: rd,collate,namespace
+
```

Comment:
```suggestion
```

Reply:
@cjyetman I don't know why but RStudio keeps on changing it on my machine and adding the space..

Reply:
whoops, I though @Antoine-Lalechere's machine added the space, so I suggested removing it 🤷🏻

Reply:
no, I guess it was the other way around